infra/discovery/src/listener/listener.js

```diff
@@ -66,7 +66,7 @@ const config = {
   // Unique id. Used to differentiate between the several listeners instances
   // that may run concurrently (ex: main vs webhook vs re-indexing).
   listenerId: args['--listener-id'] || process.env.LISTENER_ID || 'main',
-  // Notification web hook URL.
+  // Notification web hook URL. e.g. http://localhost:3456/events
   webhook: args['--webhook'] || process.env.WEBHOOK,
```

Comment:
should we rename this param to `notificationWebhook`? It would seem more natural to me. Don't know what if that kind of change would require some `GoogleCloud` settings update.

Reply:
I agree. It even took me a while to be sure this was the particular webhook related to notifications. I'll look around to find places where this param/env vars are used. 
@tomlinton - Let me know if there are hiding places where I should look.

Reply:
For what it's worth, that wasn't the original intent of the webhook. I think notifications was just the first consumer of it after Discord, which ended up becoming its own param. 🙂

Reply:
@sparrowDom : changed name here: https://github.com/OriginProtocol/origin/commit/a9c8705343cd8157b5cdcdf4e3bd22e3d0bdac8d
Was torn, but went with `notificationsWebhook` (plural) to match the name of our package/directory.